docs/log-into-techpass-portal.md

```diff
@@ -0,0 +1,30 @@
+# Log in to TechPass Portal
```

Comment:
Pls remove these changes as i have clubbed the log in to TechPass portal and edit profile content.